src/components/elements/MenuLink.js

```diff
@@ -4,18 +4,24 @@ import { Link } from 'react-router-dom'
 import MenuItem from 'material-ui/MenuItem'
 import ActionDelete from 'material-ui/svg-icons/action/delete'
 
-export const MenuLink = (type, id, onClick, dispatch) => {
+export const MenuLink = (type, id, title, onClick, dispatch) => {
   var typeRoute = `/${type}s/${id}`
   return (
     <MenuItem
       key={`${type}-${id}`}
       containerElement={<Link to={typeRoute} />}
-      primaryText={`${type} ${id}`}
+      primaryText={ticketPrimaryText(type, id, title)}
       rightIcon={<ActionDelete onClick={() => dispatch(onClick(id))} />}
     />
   )
 }
 
+const ticketPrimaryText = (type, id, title) => (
+  <div className='issue-title-text'>
+    {`${type}-${id}: ${title}`}
```

Comment:
I don't think the type needs to be shown to the user. What other types apart from tickets do we expect to display here?